src/test/kotlin/lotto/domain/LottoTest.kt

```diff
@@ -0,0 +1,41 @@
+package lotto.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class LottoTest {
+    @Test
+    fun `로또 번호의 개수가 6개가 넘어가면 예외가 발생한다`() {
+        assertThrows<IllegalArgumentException> {
+            Lotto(listOf(1, 2, 3, 4, 5, 6, 7))
+        }
+    }
+
+    @Test
+    fun `로또 번호에 중복된 숫자가 있으면 예외가 발생한다`() {
+        assertThrows<IllegalArgumentException> {
+            Lotto(listOf(1, 2, 3, 4, 5, 5))
+        }
+    }
+
+    @Test
+    fun `returns matched count when given winning numbers`() { // todo: improve
+        val lotto = Lotto(listOf(1, 2, 3, 4, 5, 6))
+        val winningNumbers = listOf(1, 2, 3, 4, 5, 6).map { LottoNumber(it) }
+
+        val matchedCount = lotto.match(winningNumbers)
+
+        assertThat(matchedCount).isEqualTo(6)
+    }
```

Comment:
```suggestion
        val matchedCount = lotto.match(winningNumbers)

        assert(matchedCount == 6)
    }
```

you can also use like this

Reply:
I think you should also test the bonus number since this one might be  more tricky